guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -1,95 +1,133 @@
 [id="configuring-repositories_{context}"]
 = Configuring repositories
 
-Use these procedures to enable the repositories required to install {ProductName}.
+ifdef::orcharhino[]
+Ensure the repositories required to install {ProductName} are enabled on your {EL} host.
+endif::[]
+
+ifdef::foreman-el,katello[]
+.Prerequisites
+* Ensure that you have access to your {EL} `BaseOS` and `AppStream` repositories.
```

Comment:
I'm not sure I'd mention this as a prerequisite. While technically correct, this is the default and AFAIK any EL is rather useless without them.

Reply:
I agree with this view of yours. I think we should reword it from `access` to `enabled`?

What I read in blogs is Admins intentionally keep those repos disabled because:
- They want a completely disconnected system and install packages from local sources.
- They do not want to update/upgrade to maintain compatibility.
and many more.

Hence, they need to enable those repos to install the Project. Also, we are explicitly mentioning for RHEL to enable those repos (even though they are enabled by default in most cases).

I might be wrong here in some cases and I am open to your ultimate view :+1:

Reply:
> They want a completely disconnected system and install packages from local sources.

In upstream this is a scenario we don't support today. If we did, it should be in the disconnected installation guide.

> * They do not want to update/upgrade to maintain compatibility.
>       and many more.

We explicitly tell them the packages should be the latest. IMHO this isn't supported either.

> Also, we are explicitly mentioning for RHEL to enable those repos (even though they are enabled by default in most cases).

That's because `subscription-manager repos --disable "*"` disables them.

> I might be wrong here in some cases and I am open to your ultimate view 👍

I wonder what others think, but I'd feel confident in relying on users to have the default repositories enabed.

Reply:
regarding default repos: Yes, a system without repos is not ideal. Maybe we can add a prerequisite such as "Ensure your host has access to the BaseOS and AppStream repositories for {EL}."? I wonder how we handle this in other procedures :shrug: